### PR TITLE
feat: Add enablePictureInPicture option

### PR DESCRIPTION
**Pull request by mister-ben**
Documents `enableDocumentPictureInPicture` option for videojs/video.js/pull/8113

### CODE REVIEW

File: src/mdx-pages/guides/options.mdx [hunk 1]
```diff
@@ -193,6 +193,26 @@ See the file `sandbox/responsive.html.example` for an example of a responsive pl
...
+
+This does not disable the document picture-in-picture mode which allows the player element to put into a PiP window.
+
+### `enableDocumentPictureInPicture`
+
+> Type: `boolean`
+
+If `true`, the [documentPictureInPicture API](https://developer.chrome.com/docs/web-platform/document-picture-in-picture/) will be used for picture in picture, if available. Defaults to `false`, but may default to `true` when the feature has become established.
+
+Currently this is avialable in Chrome 111+ as an origin trial.
```
Comment (Chocobozzz) [inline src/mdx-pages/guides/options.mdx:212]: Typo: `avialable` :)

File: src/mdx-pages/guides/options.mdx [hunk 2]
```diff
@@ -193,6 +193,28 @@ See the file `sandbox/responsive.html.example` for an example of a responsive pl
 
 This option is inherited from the [`Component` base class](#component-options).
 
+### `disablePictureInPicture`
+
+> Type: `boolean`
+
+If `true`, switching the video element into picture-in-picture is disabled. Defautl is `false`.
```
Comment (wseymour15) [inline src/mdx-pages/guides/options.mdx:200]: Typo: "Default"

File: src/mdx-pages/guides/options.mdx [hunk 3]
```diff
@@ -193,6 +193,28 @@ See the file `sandbox/responsive.html.example` for an example of a responsive pl
...
+
+This does not disable the document picture-in-picture mode which allows the player element to put into a PiP window.
+
+### `enableDocumentPictureInPicture`
+
+> Type: `boolean`
+
+_Since 8.3.0_
+
+If `true`, the [documentPictureInPicture API](https://developer.chrome.com/docs/web-platform/document-picture-in-picture/) will be used for picture in picture, if available. Defaults to `false`, but may default to `true` when the feature has become established.
```
Comment (wseymour15) [inline src/mdx-pages/guides/options.mdx:212]: nit: I think we want to include hyphens in "picture-in-picture", just to keep the doc consistent.

File: src/templates/HomeTemplate.jsx [hunk 4]
```diff
@@ -23,6 +24,10 @@ const IndexPage = props => {
         keywords={['HTML5 video', 'player', 'hls', 'adaptive-bitrate']}
         description="The world's most popular open source HTML5 player framework"
       />
+      <Helmet>
+        {/* Oriign trial for documentPictureInPicture */}
```
Comment (wseymour15) [inline src/templates/HomeTemplate.jsx:28]: typo: "Origin"

**Review by wseymour15 — CHANGES_REQUESTED** [review]
Looks good! Approved once the typos and nits are updated.

**Review by wseymour15 — APPROVED** [review]

**Pull request merged**